<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HistoryDlg</class>
 <widget class="QDialog" name="HistoryDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>512</width>
    <height>407</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>11</pointsize>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QFrame" name="bg_frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>512</width>
     <height>411</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="ZsylChart" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>90</y>
      <width>440</width>
      <height>200</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="nextButton">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>310</y>
      <width>79</width>
      <height>79</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normalon>:/pic/右.png</normalon>
     </iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>79</width>
      <height>79</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="preButton">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>310</y>
      <width>79</width>
      <height>79</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normalon>:/pic/左.png</normalon>
     </iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>79</width>
      <height>79</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="oneButton">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>314</y>
      <width>90</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/pic/button_bg.png);</string>
    </property>
    <property name="text">
     <string>获取历史</string>
    </property>
   </widget>
   <widget class="QPushButton" name="closeButton">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>314</y>
      <width>90</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/pic/button_bg.png);</string>
    </property>
    <property name="text">
     <string>关 闭</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>185</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/pic/button_bg.png);
color: rgb(0, 0, 0);</string>
    </property>
    <property name="duplicatesEnabled">
     <bool>false</bool>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
    <property name="modelColumn">
     <number>0</number>
    </property>
    <item>
     <property name="text">
      <string>转速</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>压力</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>电流</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>电压</string>
     </property>
    </item>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ZsylChart</class>
   <extends>QWidget</extends>
   <header>zsylchart.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
